requirements.txt

```diff
@@ -1,5 +1,6 @@
 git-lfs
-natcap.invest
+#natcap.invest
+git+https://github.com/natcap/invest.git@release/3.12#egg=natcap.invest
```

Comment:
> Is this part of our normal workflow for releases?

I would argue yes, even though we haven't done this kind of thing for some time and it hasn't been necessary in the UG until now because of the UG's relatively recent dependency on `natcap.invest`.